ios/22/_sentry_armv7.crash

```diff
@@ -0,0 +1,31 @@
+OS Version: iOS 9.3.5 (13G36)
+Report Version: 104
+
+Exception Type: EXC_BREAKPOINT
+Crashed Thread: 0
+
+Application Specific Information:
+Exception 6, Code 1, Subcode 0
+
+Thread 0 name:
+Thread 0 Crashed:
+0   CrashLibiOS                     0x189fe8            @objc CRLCrashSwift.crash() -> ()
+1   CrashLibiOS                     0x189fe8            CRLCrashSwift.crash() -> () (CRLCrashSwift.swift:36)
```

Comment:
Reporting the same method twice with different lines and the method doesn't call itself recursively results in this being not complete. So this frame should be marked as invalid frame and the result for 32bit here is incomplete.